web/views.py

```
@@ -4721,71 +4742,503 @@ def delete_team_goal(request, goal_id):
+            customization.has_windows = data.get("has_windows", customization.has_windows)
+            customization.has_bookshelf = data.get("has_bookshelf", customization.has_bookshelf)
+            customization.has_clock = data.get("has_clock", customization.has_clock)
+            customization.has_carpet = data.get("has_carpet", customization.has_carpet)
+
+            customization.save()
+            return JsonResponse({"status": "success"})
+        except json.JSONDecodeError:
+            return JsonResponse({"status": "error", "message": "Invalid JSON data"}, status=400)
+        except Exception as e:
+            # Log the detailed exception for debugging
+            logger.exception("Error in virtual_classroom_detail customization: %s", str(e))
```

Ruff (0.14.0)

Redundant exception object included in `logging.exception` call

(TRY401)